_summaries/kingma2014adam_2.md

```diff
@@ -1,19 +1,28 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: kingma2014adam
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+  * Introduces the Adam (Adaptive Moment Estimation) algorithm.
+    * First-order gradient-based optimization of stochastic objective functions
+  * Algorithm based on adaptive estimates
```

Comment:
It would be nice if the reviewer can elaborate more on why `adaptive estimates` is better, e.g., avoid gradient explosion.